src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -114,7 +134,7 @@ public AttachCaseCallbackService(CcdApi ccdApi) {
                 hasServiceNameInCaseTypeId(exceptionRecord),
                 caseReferenceTypeValidation,
                 caseReferenceValidation,
-                hasAnId(exceptionRecord),
+                hasId(exceptionRecord),
```

Comment:
please explain

Reply:
This is because I need CaseDetails in the event, not just an ID

Reply:
I still don't understand

Reply:
When SUPPLEMENTARY_EVIDENCE is attached to an existing case we need only ID of this case. But for SUPPLEMENTARY_EVIDENCE_WITH_OCR we need the whole case object because it is sent to the UPDATE endpoint of a client service along with the exception record

Reply:
So it's not a rename then.
Do we now have both `hasAnId` and `hasId` methods and one of them returns and ID and the other one an exception record?

Reply:
Yes, exactly